src/info.js

```diff
@@ -37,7 +37,7 @@ const getFunctionInfo = async function (srcPath) {
     return {}
   }
 
-  if (extension === '.zip' || extension === '.js') {
```

Comment:
Should we add some automated tests to cover the new file extension(s)?

Reply:
To revisit once we add support for new extensions.